GS_example.py

```diff
@@ -17,9 +17,13 @@
 if __name__ == "__main__":
     params = PropagationParams.get_example_propagation_data()
 
+    # Choose proper propagation parameters
     params.sigma = 4
     params.matrix_size = 256
     params.pixel = 0.9
```

Comment:
I assume this is length of the pixel side in `um`? If so I would add units, 

BTW @bbrozyna maybe you know - Isn't there some form of extension to have unit literals in code, like:
```python
pixel = 0.9 um #0.9e-6
wavelength = 632.8 nm #632.8e-9
```

Reply:
Sure, but I dont think it's a scope of this PR. I suggest opening a separate ticket

Reply:
https://lwn.net/Articles/900739/

Reply:
Great!! I love it, let's try it. But we need to sync to figure out our base unit then and define it

Reply:
https://github.com/bbrozyna/light-propagation/issues/16